lib/infomon/activespell.rb

```diff
@@ -0,0 +1,107 @@
+module ActiveSpell
+  #
+  # Spell timing true-up (Invoker and SK item spells do not have proper durations)
+  # this needs to be addressed in class Spell rewrite
+  # in the meantime, this should mean no spell is more than 1 second off from
+  # Simu's time calculations
+  #
+
+  def self.get_spell_info
+    puts "spell update requested\r\n" if $infomon_debug
+    @update_spell_durations = XMLData.active_spells
```

Comment:
you should not use `@` variables like this as they are scoped to the entire class, instead this method should return `[update_spell_durations, update_spell_names]` so it is more pure